napari/_qt/widgets/qt_viewer_buttons.py

```diff
@@ -97,54 +98,42 @@ def __init__(self, viewer):
         super().__init__()
 
         self.viewer = viewer
+        action_manager.context['viewer'] = viewer
+
         self.consoleButton = QtViewerPushButton(
             self.viewer,
             'console',
             trans._(
-                "Open IPython terminal ({shortcut})",
-                shortcut=Shortcut('Control-Shift-C').platform,
+                "Open IPython terminal",
             ),
         )
         self.consoleButton.setProperty('expanded', False)
         self.rollDimsButton = QtViewerPushButton(
             self.viewer,
             'roll',
-            trans._(
-                "Roll dimensions order for display ({shortcut})",
-                shortcut=Shortcut('Control-E').platform,
-            ),
-            lambda: self.viewer.dims._roll(),
         )
+
+        action_manager.bind_button('napari:roll_axes', self.rollDimsButton)
```

Comment:
Yeah, right now we have to create the button, and bind, but we could also do a `action_manger.get_button_for(....)` and it would return a button instance; though that was mixing a bit of qt logic in action manager.

Reply:
yeah, i think fine to leave as is now for this PR, and then can think in future about adding convenience methods like that. I do like having the qt stuff separate